hazelcast-wm/src/main/java/com/hazelcast/web/ClusteredSessionService.java

```diff
@@ -104,18 +103,14 @@ public ClusteredSessionService(FilterConfig filterConfig, Properties properties,
 
         es.scheduleWithFixedDelay(new Runnable() {
             public void run() {
-                while (ensureInstanceRunning) {
-                    try {
-                        ensureInstance();
-                    } catch (Exception ignored) {
-                        if (ignored instanceof InterruptedException) {
-                            ensureInstanceRunning = false;
-                        }
-                        EmptyStatement.ignore(ignored);
-                    }
+                try {
+                    ensureInstance();
+                } catch (Exception ignored) {
+                    EmptyStatement.ignore(ignored);
                 }
             }
         }, 0, CLUSTER_CHECK_INTERVAL, TimeUnit.SECONDS);
```

Comment:
`CLUSTER_CHECK_INTERVAL = 1000L` , so every 1000 seconds is seems too much. And probably test is failed due to this interval... Maybe we can decrease this interval.